providers/mod.ts

```diff
@@ -24,6 +25,7 @@ providers.addMultiple(
 	DeezerProvider,
 	iTunesProvider,
 	SpotifyProvider,
+	OtotoyProvider,
```

Comment:
I would put this before or after mora. This would keep the HTML extracting providers at the end of the list.